composer.json

```diff
@@ -58,15 +58,15 @@
         "sonata-project/doctrine-orm-admin-bundle": "^3.0",
         "sonata-project/formatter-bundle": "^3.2",
         "sonata-project/seo-bundle": "^2.1",
-        "symfony/phpunit-bridge": "^2.7 || ^3.0"
+        "symfony/phpunit-bridge": "^3.3"
     },
     "suggest": {
         "liip/imagine-bundle": "^0.9",
         "rackspace/php-opencloud": "^1.6",
         "sonata-project/classification-bundle": "If you want to categorize your media items.",
         "sonata-project/doctrine-orm-admin-bundle": "^3.0",
         "sonata-project/seo-bundle": "^2.1",
-        "symfony/monolog-bundle": "^2.4",
+        "symfony/monolog-bundle": "^2.8",
```

Comment:
This version does not affect anything, we should change it for a description like: "If you want to log exceptions produced when dealing with images"

Reply:
Right! IMO, I should add a conflict with `"symfony/monolog-bundle": "<2.8 || >=3.0 <3.2"`. Is it the correct way to do it?

Reply:
Why conflict? If we detect something that makes us unable to work with some version, lets do it. But I think this is not the case here

Reply:
> If we detect something that makes us unable to work with some version, lets do it.

That's exactly what we agreed to not do when we decided to drop support of SF < 2.8.
Currently, monolog is only an optionnal dependency, and we use it when it is installed.

So, why should we not handle optionnal dependencies like "normal" ones?

Reply:
optional dependencies might be required in the project to do something else entirely

Reply:
But they we can't disable their usage if installed in this case, so we should indeed conflict.

Reply:
If you want to keep that conflict the way it is, fine by me, but monolog-bundle does not follow same version as symfony does, I am not sure on what is based that conflict requirement.

From monolog readme:

```
As of v2.4.0 of the bundle, the release cycle is de-synchronized from the framework's. It means you can just require "symfony/monolog-bundle": "~2.4" in your composer.json and Composer will automatically pick the latest version of the bundle that works with your current version of Symfony. The minimum version of Symfony2 for this workflow is 2.3.0.
```

Reply:
I didn't know that this bundle does not follow the same release cycle as symfony.
I changed the conflict to `< 2.4`.